sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
+watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
+termed deep learning, seem like a solution to the challenge presented by the
+growth of data in biomedicine. Perhaps these algorithms could identify the
+biological "cats" hidden in our data - the patterns that exist but that we don't
+know to look for - and could act on them.
 
-### If this happens, is deep learning required for any of it? Are we any closer
-### because of the advent of deep learning?
+Deep learning has transformed image analysis, but researchers' initial forays
+into the use of these techniques in biomedicine have been relatively limited.
+There are certainly numerous promising examples, but we have not yet seen the
+massive convergence on these approaches that occurred in the field of image
+analysis. In this review, we discuss whether this is simply a matter of time or
+if there are unique challenges posed by biomedical data that render deep
+learning methods more challenging or less fruitful to apply.
 
-*"Categorize" and "treat" sound a bit like PMI goals. Another way to think about
-this would be: do we think that deep learning will make much of a difference
-for the precision medicine initiative (PMI)?*
+### What is deep learning?
 
-### General deep learning introduction
+Deep learning is built on a biologically-inspired approach from machine learning
+termed neural networks. Each neuron in a computational neural network, termed a
+node, has inputs, an activation function, and outputs. Each value from the
+inputs is usually multiplied by some weight and combined and summarized by the
+activation function. The value of the activation function is then multiplied by
+another set of weights to produce the output `TODO: we probably need a figure
+here - I see no way that we don't include this type of description in our paper,
+despite the fact that it's been done tons of times before.` These neural
+networks are trained by identifying weights that produce a desired output from
+some specific input.
 
-*Definitions, specific architectures, etc.  We may want to clarify what we mean
-by "deep" learning when most methods use few hidden layers.*
+Neural networks can also be stacked. The outputs from one can be used as inputs
+to another. This process produces a stacked or multi-layer neural network. The
+multi-layer neural network techniques that underlie deep learning have a long
+history. Multi-layer methods have been discussed in the literature for more than
+five decades [@doi:10.1103/RevModPhys.34.135]. Given this context, it's
+challenging to consider "deep learning" as a new advance, though the term has
+only become widespread to describe analysis methods in the last decade. For the
+purposes of this review, we identify deep learning approaches as those that use
+multi-layer neural networks to construct complex features from large-scale
+datasets. `TODO: somehow, I feel like we should work in some of the early
```

Comment:
I would stay away from the history, in part because I don't know all of the details well enough to give appropriate credit.  There are also very early examples of neural networks in biology that we don't necessarily want to discuss (e.g. [secondary structure prediction](https://www.ncbi.nlm.nih.gov/pubmed/3172241) in 1988).  We could refer to something like http://www.deeplearningbook.org/ to cover the history of neural networks instead of doing it ourselves.

I do think we need to clarify "deep" here, which you did.  To me at least, the term has come to represent a set of strategies for constructing and training neural networks even if there are few layers.